docs/source/admin/traffic_portal/usingtrafficportal.rst

```diff
@@ -602,6 +602,8 @@ User management includes the ability to (where applicable):
 - update an existing user
 - view :term:`Delivery Service`\ s visible to a user
 
+.. Note:: If OAuth is enabled, creating/updating a user here will update the user's roles but the user needs to be created/updated with the OAuth provider as well.
```

Comment:
a user just has one role

Reply:
Also, `Role` is a `term`

Reply:
done. also changing 'updating' to 'deleting' since right now oauth only deals with user authentication and changing the user's role will not have any effect on it, so only creates/deletes need to be done in the oauth provider too